cmd/list.go

```diff
@@ -14,7 +14,7 @@ import (
 
 var createdOnly bool
 var listAll bool
-var componentName string
+var componentNames []string
```

Comment:
should there be a space between componentNames and []string?

Reply:
I changed from a single value to multiple possible; this is valid go syntax for the array.